### PR TITLE
feat(styx-cli): operational wrapper for the Audience Growth Engine

### PR DESCRIPTION
**Pull request by 4444J99**
Adds packages/styx-cli, the friendly face of the engine. Wraps @styx/audience-engine for Styx/Jessica defaults, plus a metric logger for the weekly review.

Two commands:
- `styx plan-week [--start <iso>] [--output <md>]`: runs the engine with the bundled Styx config and writes the 30-day plan
- `styx log-week --week <YYYY-WNN> --followers <N> --email <N> [--opens <%>] [--waitlist <N>]`: appends to metrics.csv for the weekly review

4 vitest tests pass. CLI smoke-tested. isoWeekString uses UTC math to avoid timezone drift.

### CODE REVIEW

File: packages/styx-cli/src/cli.ts [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  const dayNr = (d.getDay() + 6) % 7;
+  target.setDate(target.getDate() - dayNr + 3);
+  const firstThursday = target.valueOf();
+  target.setMonth(0, 1);
+  if (target.getDay() !== 4) {
+    target.setMonth(0, 1 + ((4 - target.getDay() + 7) % 7));
+  }
+  const weekNumber =
+    1 + Math.ceil((firstThursday - target.valueOf()) / 604800000);
+  return `${d.getFullYear()}-W${String(weekNumber).padStart(2, "0")}`;
```
[the remark this answers is not in the document]
Comment (4444J99) [reply packages/styx-cli/src/cli.ts]: Fixed. Replaced local `isoWeekString` (which used `d.getFullYear()` — Gregorian year) with the shared UTC-based `isoWeekString` from `./iso-week.js` that computes the ISO year from the shifted Thursday. 4 existing tests pass, including New Year boundary cases.

File: packages/styx-cli/src/cli.ts [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+}
+
+function findDefaultConfig(): string {
+  // The default Styx config is in the audience-engine package's templates.
+  // We look for it relative to the current working directory first, then
+  // in the package's node_modules.
+  const candidates = [
+    resolve("packages/audience-engine/templates/styx-instance.yaml"),
+    resolve("node_modules/@styx/audience-engine/templates/styx-instance.yaml"),
+    resolve("../audience-engine/templates/styx-instance.yaml"),
```
[the remark this answers is not in the document]
Comment (4444J99) [reply packages/styx-cli/src/cli.ts]: Fixed. Uses `createRequire(require.resolve("@styx/audience-engine/package.json"))` to resolve the config path from the installed module, not from `process.cwd()`. Works regardless of where the operator runs the CLI.

File: packages/styx-cli/src/cli.ts [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+  } catch (err) {
+    console.error(
+      `✗ Config invalid: ${err instanceof Error ? err.message : String(err)}`,
+    );
+    return 1;
+  }
+  const startDate = args.start
+    ? parseDateNoTimezoneDrift(args.start)
+    : undefined;
+  const plan = generatePlan(config, startDate);
```
[the remark this answers is not in the document]
Comment (4444J99) [reply packages/styx-cli/src/cli.ts:89]: Fixed. Added `Number.isNaN(startDate.getTime())` guard after `parseDateNoTimezoneDrift`, returns exit code 1 with error message before reaching `generatePlan`.

File: packages/styx-cli/src/cli.ts [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+      console.error("Error: log-week requires --week, --followers, --email");
+      printUsage();
+      return 2;
+    }
+    return cmdLogWeek({
+      week: args.week,
+      followers: Number(args.followers),
+      email: Number(args.email),
+      opens: args.opens !== undefined ? Number(args.opens) : undefined,
+      waitlist: args.waitlist !== undefined ? Number(args.waitlist) : undefined,
```
[the remark this answers is not in the document]
Comment (4444J99) [reply packages/styx-cli/src/cli.ts]: Fixed. Parses `--followers`, `--email`, `--opens`, `--waitlist` to numbers before `cmdLogWeek` and rejects non-finite values with exit code 2 + descriptive error before any CSV write.
